src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.io.*;
```

Comment:
You may wish to consider [explicitly importing](https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements) the classes that you wish to use here to increase clarity of what classes are to be maintained here.